src/components/PopUpWithConfirm.js

```diff
@@ -0,0 +1,29 @@
+import PopUp from "./Popup.js";
+
+export default class PopUpWithConfirm extends PopUp{
+    constructor(popupSelector) {
+        super({popupSelector});
+        this._popupForm = this._popupElement.querySelector(".modal__form");
+        this._cardElement = document.querySelector(".card");
```

Comment:
`        this._cardElement = document.querySelector(".card");
`

Please, delete it

You don't use it and you can't use it here